Documentation/freebsd-instructions.md

```diff
@@ -24,13 +24,35 @@ Install the following packages for the toolchain:
 - libunwind
 - gettext
 
-Note: LLDB is not yet available in the pkg repository on FreeBSD. The LLDB plugin is currently not being build on FreeBSD. Instructions for this will be added later.
-
 To install the packages you need:
 
-```janhenke@freebsd-frankfurt:~ % sudo pkg install bash cmake clang35 libunwind gettext```
+``janhenke@freebsd-frankfurt:~ sudo pkg install bash cmake clang35 libunwind gettext```
```

Comment:
On a few projects I've worked on, using single tickets for inline/single lines helped avoid issues with some highlighters / md implementations.  https://github.com/dotnet/coreclr/pull/900